pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -148,16 +149,24 @@ func lookupContentTreeStatusAny(ctx *volumemgrContext, key string) *types.Conten
 	return nil
 }
 
-func getAllAppContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
-	log.Tracef("getAllAppContentTreeStatus")
-	pub := ctx.publication(types.ContentTreeStatus{}, types.AppImgObj)
-	contentIDAndContentTreeStatusIntf := pub.GetAll()
+func getAllContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
+	var (
+		pub                               pubsub.Publication
+		contentIDAndContentTreeStatusIntf map[string]interface{}
```

Comment:
Shouldn't this be a map[string]*types.ContentTreeStatus?

Oh, I see this is just the local result from GetAll. no need to declare that.
And it is easier to read if there are several similar sounding long variable names.
for key, item := range pub.GetAll() {
does the job and is much easier to read.

Reply:
Changed.